components/embedder_support/android/java/src/org/chromium/components/embedder_support/view/ContentViewRenderView.java

```diff
@@ -95,6 +95,9 @@ public void surfaceCreated(SurfaceHolder holder) {
                 getSurfaceView().setVisibility(getSurfaceView().getVisibility());
 
                 onReadyToRender();
+
```

Comment:
Quick drive by comment - we should `#ifdef` or feature-gate all those changes that change behavior that significantly. 

Unmodified Chromium builds behavior should not change

What is the correct way to `#ifdef` ? I'm not sure, but lets at least pick something, even temporary, to get code landed.

Reply:
talked about this on chat : https://chromium.googlesource.com/chromium/src/+/HEAD/docs/how_to_add_your_feature_flag.md

this seems like the way to go.